flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -924,6 +936,15 @@ class TaskManager(
     else {
       libraryCacheManager = Some(new FallbackLibraryCacheManager)
     }
+
+    metricsRegistry = new FlinkMetricRegistry(GlobalConfiguration.getConfiguration);
```

Comment:
I think it's better to use `config.configuration`. At some point in time we might get around to removing the `GlobalConfiguration`.